### ISSUE TITLE
for dense voxel visualization

### ISSUE THREAD

**Issue by pullitdown** [NVlabs/VoxFormer#37]
awosome work! The visualization effect of the dense voxel map in the document is very good, but when I use the output of the model for voxel visualization, my gridmap is not as dense as in the document. like car voxel , it would not occupy the space from car top to groud ,but single layer grid in car top space. Could you tell me how to visualize the voxel better? 
using command
./tools/dist_test.sh ./projects/configs/voxformer/voxformer-S.py ./ckpts/voxformer-S/miou12.35_iou44.02_epoch_14.pth 4 
and collecting  y_pred(np.array shape 256,256,32).
do i need change my  model head？
thanks a loooot!

my grid
https://lingdongfangcheng.feishu.cn/file/VaDOb6kqHoS2ywxg5tXcfVIMnIg?from=from_copylink

**Comment by pullitdown** [comment]
solved  in https://github.com/NVlabs/VoxFormer/issues/13
i remap the label from prediction to kitti config and use kiiti dataset visualize script to visualization.

**Comment by max-monastirsky** [comment]
@pullitdown How do you remap the labels? Could you elaborate the visulalization procedure?

**Comment by SZUshenyan** [comment]
> solved in #13 i remap the label from prediction to kitti config and use kiiti dataset visualize script to visualization.
Hello, I used the following code visualization, but the generated voxel map is strange, can you provide your visualization code? Thank you!
```
`import os
import numpy as np

# 加载保存的 numpy 数组
y_pred = np.load('y_pred.npy')

# 输出加载的数组及其形状
print("Loaded numpy array:")
print(y_pred)
print("Shape of loaded numpy array:", y_pred.shape)

# 修改数组的形状和数据类型
new_shape = (256, 256, 32)
y_pred_reshaped = y_pred.reshape(new_shape).astype(np.uint16)

# 创建文件夹用于保存 .label 文件
output_folder = '/home/sy/semantic-kitti-api/kitti/dataset/sequences/00/voxels'
os.makedirs(output_folder, exist_ok=True)

# 保存每个标签为 .label 文件
for i, label in enumerate(y_pred_reshaped):
    # 生成文件名，例如 000000.label
    filename = f'{i:06d}.label'
    filepath = os.path.join(output_folder, filename)
    # 保存标签为 .label 文件
    label.tofile(filepath)
```
![Uploading 573514c85c4e81bb0caae555d43d549.png…]()

**Comment by SZUshenyan** [comment]
![573514c85c4e81bb0caae555d43d549](https://github.com/NVlabs/VoxFormer/assets/152782550/17e78068-48a9-4611-94d1-afab194db3e8)
